yt_dlp/extractor/tiktok.py

```diff
@@ -6,16 +6,15 @@
 import time
 
 from .common import InfoExtractor
-from ..compat import compat_urllib_parse_unquote, compat_urllib_parse_urlparse
 from ..networking import HEADRequest
+from ..compat import compat_urllib_parse_urlparse
```

Comment:
revert unnecessary/unrelated formatting change
```suggestion
from ..compat import compat_urllib_parse_urlparse
from ..networking import HEADRequest
```